secureli/container.py

```diff
@@ -12,22 +12,16 @@
 from secureli.repositories.secureli_config import SecureliConfigRepository
 from secureli.repositories.settings import SecureliRepository
 from secureli.modules.shared.resources import read_resource
-from secureli.modules.language_analyzer.language_analyzer_services.git_ignore import (
-    GitIgnoreService,
-)
-from secureli.modules.language_analyzer.language_analyzer_services.language_analyzer import (
-    LanguageAnalyzerService,
-)
-from secureli.modules.language_analyzer.language_analyzer_services.language_support import (
-    LanguageSupportService,
+from secureli.modules.language_analyzer.language_analyzer_services import (
+    git_ignore,
+    language_analyzer,
+    language_support,
+    language_config,
 )
```

Comment:
I am not sure that is meets the AC's in terms of making imports pythonic, but if I try `from secureli.modules.language_analyzer import language_analyzer_services` the imports break.

Reply:
would from  from `secureli.modules.language_analyzer import LanguageAnalyzerService` work?